src/Transformations.jl

```diff
@@ -261,6 +269,15 @@ function transform!(tn::TensorNetwork, config::ColumnReduction)
     return tn
 end
 
+"""
+    SplitSimplification <: Transformation
+
+Reduces the rank of some tensors in the `TensorNetwork` by splitting some tensors into two. It uses teh singular value decomposition to identify the tensors that can be split without increasing the maximum rank of the network.
```

Comment:
> ... by splitting some tensors into two.
Can you rephrase this?

```suggestion
Reduce the rank of some tensors in the `TensorNetwork` by splitting some tensors into two. It uses teh singular value decomposition to identify the tensors that can be split without increasing the maximum rank of the network.
```